common/src/Microsoft.Azure.IIoT.Diagnostics.AppInsights/src/Default/ApplicationInsightsTelemetryInitializer.cs

```diff
@@ -18,15 +18,15 @@ public class ApplicationInsightsTelemetryInitializer : ITelemetryInitializer {
         /// Constructor
         /// </summary>
         /// <param name="processIdentity"></param>
-        public ApplicationInsightsTelemetryInitializer(IProcessIdentity processIdentity) {
+        public ApplicationInsightsTelemetryInitializer(IProcessIdentity processIdentity = null) {
```

Comment:
@marcschier, do we need a default constructor with `null` as `processIdentity` ? I can see only one initialization of `ApplicationInsightsTelemetryInitializer` (in `AppInsightsContainerBuilderEx` class) and that one even checks that `processIdentity` is not `null`.

Reply:
It is also injected and then process identity might be missing.